parser_library/src/workspaces/workspace.cpp

```diff
@@ -319,6 +322,20 @@ bool workspace::load_and_process_config()
                 // else ignore, publish warning
             }
         }
+        if (asm_options_json.size() != 0)
+        {
+            std::map<std::string, std::string> asm_options;
+
+            if (asm_options_json.count("SYSPARM"))
```

Comment:
I am thinking that maybe we should just copy all key-string pairs from the json dictionary to the map? That way, adding new options in the future will not require change in this code.

Reply:
Additional parameters may need validation etc.